samples/getting_started.ipynb

```diff
@@ -0,0 +1,616 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting Started with cuDF"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading data into a GPU DataFrame (GDF)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cudf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Loading data into a GPU DataFrame\n",
+    "\n",
+    "It's easy to load almost any sort of data (json, csv, etc) into a GPU DataFrame. Ex (csv import from disk):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# read data from csv file into pandas dataframe\n",
+    "gdf = cudf.read_csv('data/ipums/ipums_easy.csv.gz', compression='gzip')"
```

Comment:
`compression` is not yet a supported parameter of the `read_csv` function.

Data should be decompressed before this.

Reply:
We also don't support column name inference or type inference quite yet so you need to define the column names and dtypes and pass them into this call.